### ISSUE TITLE
[Perf][Android] `VirtualizingPanelLayout` invokes `TryCreateLine` on non-modified layout

### ISSUE THREAD

**Issue by jeromelaban** [unoplatform/uno#6899]
### Current behavior

After the changes done for https://github.com/unoplatform/uno/issues/6754, the measure pass was improved and done only once when a single item requests a measure, but invocations to TryCreateLine are still invoked for all items:

https://github.com/unoplatform/uno/blob/6471ab2b4691e81a2261309f83721a3df2b34728/src/Uno.UI/UI/Xaml/Controls/ListViewBase/VirtualizingPanelLayout.Android.cs#L1366

### Expected behavior

The performance profiling does not show any time spent measuring/adding items that are not changed in the list.

### NuGet package version(s)

3.10

### Affected platforms

Android

**Comment by MartinZikmund** [comment]
This is no longer relevant in case of Skia Android target of Uno Platform
